# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344299238 25200
#      Mon Aug 06 17:27:18 2012 -0700
# Node ID 9db33c1f6f5f785edd531af1813260cd02815d6c
# Parent  718bed882d3ecdea4007e1d53fe22ebf916d1748
Remove CPPFLAGS from AM_CPPFLAGS which causes CPPFLAGS to appear twice during compile.

* libcruft/Makefile.am, liboctave/Makefile.am, src/Makefile.am:
Remove instantiation of CPPFLAGS in AM_CPPFLAGS.

diff --git a/libcruft/Makefile.am b/libcruft/Makefile.am
--- a/libcruft/Makefile.am
+++ b/libcruft/Makefile.am
@@ -17,18 +17,17 @@
 # You should have received a copy of the GNU General Public License
 # along with Octave; see the file COPYING.  If not, see
 # <http://www.gnu.org/licenses/>.
 
 include $(top_srcdir)/build-aux/common.mk
 
 ## Search local directories before those specified by the user.
 AM_CPPFLAGS = \
-  -I../libgnu -I$(top_srcdir)/libgnu \
-  @CPPFLAGS@
+  -I../libgnu -I$(top_srcdir)/libgnu
 
 AUTOMAKE_OPTIONS = subdir-objects
 
 octlib_LTLIBRARIES = libcruft.la
 
 libcruft_la_SOURCES =
 
 nodist_libcruft_la_SOURCES =
diff --git a/liboctave/Makefile.am b/liboctave/Makefile.am
--- a/liboctave/Makefile.am
+++ b/liboctave/Makefile.am
@@ -18,18 +18,17 @@
 # along with Octave; see the file COPYING.  If not, see
 # <http://www.gnu.org/licenses/>.
 
 include $(top_srcdir)/build-aux/common.mk
 
 ## Search local directories before those specified by the user.
 AM_CPPFLAGS = \
   -I../libgnu -I$(top_srcdir)/libgnu \
-  -I$(top_srcdir)/libcruft/misc \
-  @CPPFLAGS@
+  -I$(top_srcdir)/libcruft/misc
 
 EXTRA_DIST = \
   config-ops.sh \
   mk-ops.awk \
   mx-op-inc.mk \
   mx-op-src.mk \
   mx-ops \
   smx-op-inc.mk \
diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -26,18 +26,17 @@ AM_CPPFLAGS = \
   -I../liboctave -I$(top_srcdir)/liboctave \
   -Ioctave-value -I$(srcdir)/octave-value \
   -Iparse-tree -I$(srcdir)/parse-tree \
   -Ioperators -I$(srcdir)/operators \
   -Iinterp-core -I$(srcdir)/interp-core \
   -Iinterpfcn -I$(srcdir)/interpfcn \
   -Icorefcn \
   -I. -I$(srcdir) \
-  -I../libgnu -I$(top_srcdir)/libgnu \
-  @CPPFLAGS@
+  -I../libgnu -I$(top_srcdir)/libgnu
 
 AUTOMAKE_OPTIONS = subdir-objects
 
 octlib_LTLIBRARIES = liboctinterp.la
 
 if AMCOND_BUILD_COMPILED_AUX_PROGRAMS
 bin_PROGRAMS = \
   mkoctfile \
